docs/reference/cluster-operator/examples.md

```diff
@@ -161,6 +161,77 @@ Limiting Ondat can cause malfunction for IO to Ondat volumes, therefore
 we do not currently recommend applying upper limits to resources for Ondat
```

Comment:
Maybe we should remove this, as it conflicts with the following content that provides limits.